07-advanced-git/README.md

```diff
@@ -1,3 +1,3 @@
 # Table of contents
 
-1. [`Git Log`](./07-01-git-log/README.md)
+1. [Git Log](./07-01-git-log/README.md)
```

Comment:
Changing format to be consistent with root README.md  

Can change this if we want to make the root follow this format instead

Reply:
Either is fine with me, as long as both are consistent :+1: